test/Microsoft.AspNet.Mvc.FunctionalTests/XmlAssert.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Xml.Linq;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Mvc.FunctionalTests
+{
+    /// <summary>
+    /// Xunit assertions related to Xml content.
+    /// </summary>
+    public static class XmlAssert
```

Comment:
This code has been duplicated in this project as well as in the `Microsoft.AspNet.Mvc.Xml.Test`